stable/xray/templates/xray-statefulset.yaml

```diff
@@ -500,12 +500,12 @@ spec:
         volumeMounts:
         - name: data-volume
           mountPath: "{{ .Values.xray.persistence.mountPath }}"
-        {{- with .Values.common.customVolumeMounts }}
-{{ toYaml . | indent 8 }}
-        {{- end }}
-        {{- with .Values.persist.customVolumeMounts }}
-{{ toYaml . | indent 8 }}
-        {{- end }}
+{{- if .Values.common.customVolumeMounts }}
+{{ tpl .Values.common.customVolumeMounts . | indent 8 }}
+{{- end }}
+{{- if .Values.persist.customVolumeMounts }}
```

Comment:
Please use with instead of the if statement

Reply:
@danielezer  Done